### ISSUE TITLE
README.md needs clarification

### ISSUE THREAD

**Issue by rogueeve575** [solosky/pixl.js#296]
I'm not sure if this is an "issue" or where it should go exactly, but I have one of these devices with an OLED screen and internal battery ([looks like this](https://shop.mtoolstec.com/wp-content/uploads/2023/11/Pixl-OLED-main-1.jpg)). It's really beautiful hardware in person, I just don't actually own a Switch. A friend sent this to me under the impression it was the same as the Espruino-based Pixl.js with different firmware and I would be able to simply flash stock Espruino onto it for a project. However, I since realize that this device is not really the same as Gordon's Pixl.js.

I was wondering if anyone might be able to provide more context as to the history of why these devices are called a Pixl.js. The README.md says it's a fork, with a link back to the Espruino repo, and indeed my AmiiboLink Pixl.js does appear as a pairing option on the official Espruino IDE (but only partially connects of course) and likewise the official Amiibo webapp offers to pair to my Espruino-based Bangle.js, so there is clearly some sort of overlap, but I don't see the actual Espruino code under this repo (unless I missed it?).

Basically I'm just trying to understand more about this neat little device so I can find different ways I could run my own code on it. It's in a really pretty case which I suspect may be glued shut, so I'm investigating the other options before I bust it open and go messing about with SWD pins. Is there still any underlying Espruino left I could get a REPL on somehow? I'm also just a bit curious about the history I guess.

To make this technically an "issue", I could request "README.md needs clarification" since it mentions Espruino but for me anyway, the connection was unclear. Also the BOM page may not be complete -- since I haven't opened my device up yet I'm actually not certain what CPU this has since I didn't see it listed, although I suspect it's probably a nRF52832?

**Comment by solosky** [comment]
Hi there,  I am glad you like this device.

Indeed, it's very confuse between the Espruino and my pixl.js project. I think it is nessary to have a brief about the pixl.js project history. 

At first, I was already working on the nrf52832 to emulate amiibo, and already design a couple of prototypes, but I don't satisfy with my protoypes. And someday I found the [Pixl.js board](https://shop.espruino.com/pixljs),  I realized that the Pixl.js board is exactly what i would like to build, so I reviewed its hardware spec and trying to build a similar device but with some hardware change to make the device better for emulate amiibo. for example, added a flash chip, re-layout the PCB, etc. it results in the current LCD version hardware. 

At first, I was focusing on the firmware development and I just use the Pixl.js as the repo name because I did not come a good name on this project. However, with the new feature was implemented quickly and I still could not have a good name on this project, so I decided to continue using Pixl.js as project name. 

That is the project history. I am sorry the project name make you confused and I total agree with you.   

Regarding why not develop the firmware using Espruino, I was considering the Pixl.js was designed to emulate Amiibo and not intend to an open source creative hardware platform for hobbies.  With native C code implementation, we could benefit from small firmware footprint and compact RAM usage. 

Anyway, I still tried the Espruino on my LCD version of Pixl.js at project beginning and works well. It requires some modification from original Espruino codebase. I think I could release the Espruino fork for Pixl.js if I am avaliable. If you intrested in running Espruino on Pixl.js device, you could follow me and stay tunned.
